### ISSUE TITLE
[Playground] Show the Langium version used in the Playground

### ISSUE THREAD

**Issue by brsanthu** [eclipse-langium/langium-website#228]
Since Langium is going through many changes, it would be good to know the Langium version used in the Playground.

**Comment by msujew** [comment]
Good idea! FYI as of a few days ago, we've finally updated to 3.0, see [here](https://github.com/eclipse-langium/langium-website/pull/220).
